sw/device/examples/hello_world/hello_world.c

```diff
@@ -24,9 +24,39 @@ static dif_pinmux_t pinmux;
 static dif_spi_device_handle_t spi;
 static dif_uart_t uart;
 
+static dif_pinmux_index_t leds[] = {
+    kTopEarlgreyPinmuxMioOutIor10,
+    kTopEarlgreyPinmuxMioOutIor11,
+    kTopEarlgreyPinmuxMioOutIor12,
+    kTopEarlgreyPinmuxMioOutIor13,
+};
+
+static dif_pinmux_index_t switches[] = {
+    kTopEarlgreyPinmuxInselIob6,
+    kTopEarlgreyPinmuxInselIob7,
+    kTopEarlgreyPinmuxInselIob8,
+    kTopEarlgreyPinmuxInselIob9,
+};
+
+void configure_pinmux(void) {
+  pinmux_testutils_init(&pinmux);
+  // Hook up some LEDs.
+  for (int i = 0; i < sizeof(leds) / sizeof(leds[0]); i++) {
```

Comment:
same here and the loop below.

nit: I'm not up to date on IO/pad/toplevel discussions but it looks like the definitions for LEDs, switches, and the loops can be lifted to a device specific file or `*_util.c` file. Should we create a cleanup issue?

Reply:
I'm not sure there's significant value in doing so for the demos. Do we plan to expand beyond the USB and UART demos we have?

For tests, LEDs and switches will not be used, and tests must configure their own pins beyond the UART. We'll likely have some generated pinmux config for a given top-level, and that is on the to-do list.

Reply:
I'm not sure, we can revisit later if needed, resolving.